plugins/elgato/package.json

```diff
@@ -0,0 +1,42 @@
+{
+   "name": "@scrypted/elgato",
+   "version": "0.1.0",
+   "description": "An Elgato KeyLight plugin for Scrypted",
+   "author": "Scrypted",
+   "license": "Apache",
+   "scripts": {
+      "prepublishOnly": "NODE_ENV=production scrypted-webpack",
+      "prescrypted-vscode-launch": "scrypted-webpack",
+      "scrypted-vscode-launch": "scrypted-deploy-debug",
+      "scrypted-deploy-debug": "scrypted-deploy-debug",
+      "scrypted-debug": "scrypted-debug",
+      "scrypted-deploy": "scrypted-deploy",
+      "scrypted-readme": "scrypted-readme",
+      "scrypted-package-json": "scrypted-package-json",
+      "scrypted-webpack": "scrypted-webpack"
+   },
+   "keywords": [
+      "elgato",
+      "keylight",
+      "scrypted",
+      "plugin"
+   ],
+   "scrypted": {
+      "name": "Elgato Controller",
+      "singleInstance": true,
+      "type": "DeviceProvider",
+      "interfaces": [
+         "DeviceProvider"
+      ]
+   },
+   "devDependencies": {
+      "@scrypted/sdk": "file:../../sdk",
```

Comment:
when you create your own repository, I think the only thing you'll need to change is this to

```
      "@scrypted/sdk": "^0.0.145"
```

Reply:
I take it that this would be a good reference for setting up as a separate repo?
https://github.com/scryptedapp/sendgrid

Reply:
also need to add a dev dependency on `webpack-bundle-analyzer` but otherwise is working after splitting it out:
https://github.com/3nth/scrypted-elgato